cs3api4lab/api/cs3apismanager.py

```diff
@@ -503,10 +503,11 @@ def create_checkpoint(self, path):
         return {'id': 'checkpoint', 'last_modified': "0"}
```

Comment:
Doesn't this create a wrong entry in the UI?

Reply:
The returned value from this endpoint is only used by front-end tests of the original components. The UI uses another endpoint for listing values and ignores the response body of the create_checkpoint  endpoint.